cmd/bosun/conf/conf.go

```diff
@@ -1326,3 +1344,11 @@ func (c *Conf) MakeLink(path string, v *url.Values) string {
 	}
 	return u.String()
 }
+
+func (c *Conf) RunModulo(alert *Alert) uint64 {
+	modulo := alert.RunEvery
+	if modulo == 0 {
```

Comment:
How about just run this at the end of `loadAlert` so that it only needs to be computed once?